cycode/cyclient/scan_client.py

```diff
@@ -20,29 +20,33 @@ def __init__(
         self.scan_cycode_client = scan_cycode_client
         self.scan_config = scan_config
 
-        self._SCAN_CONTROLLER_PATH = 'api/v1/scan'
-        self._SCAN_CONTROLLER_PATH_SCA = 'api/v1/cli-scan'
+        self._SCAN_SERVICE_CONTROLLER_PATH = 'api/v1/scan'
+        self._SCAN_SERVICE_CLI_CONTROLLER_PATH = 'api/v1/cli-scan'
 
         self._DETECTIONS_SERVICE_CONTROLLER_PATH = 'api/v1/detections'
-        self._DETECTIONS_SERVICE_CONTROLLER_PATH_SCA = 'api/v1/detections/cli'
+        self._DETECTIONS_SERVICE_CLI_CONTROLLER_PATH = 'api/v1/detections/cli'
 
         self.POLICIES_SERVICE_CONTROLLER_PATH_V3 = 'api/v3/policies'
 
         self._hide_response_log = hide_response_log
 
     def get_scan_controller_path(self, scan_type: str, should_use_scan_service: bool = False) -> str:
-        if should_use_scan_service:
-            return self._SCAN_CONTROLLER_PATH
-        if scan_type == consts.SCA_SCAN_TYPE:
-            return self._SCAN_CONTROLLER_PATH_SCA
+        if scan_type == consts.INFRA_CONFIGURATION_SCAN_TYPE:
+            # we don't use async flow for IaC scan yet
+            return self._SCAN_SERVICE_CONTROLLER_PATH
+        if not should_use_scan_service and scan_type == consts.SECRET_SCAN_TYPE:
```

Comment:
when `should_use_scan_service` is false?

Reply:
and when `https://api.cycode.com/scans/api/v1/cli-scan/Secrets/ZippedFile` is called?

Reply:
when `--report` is passed for secrets scan